frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -249,8 +249,11 @@ class Multiselect extends React.PureComponent<Props, State> {
 
                     Issue related: https://github.com/streamlit/streamlit/issues/590
                   */
-                  minHeight: "40px",
+                  minHeight: "38px",
```

Comment:
nit: I had a quick look into the PR preview. There seems to be a very slight difference in the total height of `multiselect` (=40px) and `selectbox`/`text_input` (=40.4px). Since it does not seem to be visible detectable, I think we might be fine here.

Reply:
@LukasMasuch It's probably caused by the `lineHeight` vs. `padding` issue mentioned above, so maybe if we fix that, this'll be resolved as well.

Reply:
Unforatunly, line height does not seem to fix it with multi-select. One workaround might be to actually set it to `38.4px` to have the same height. It could be another task to standardize all widgets to 40px... but that might involve a bit more effort